### PR TITLE
 react_dynamic-list-of-todos

### PR DESCRIPTION
**Pull request by IvanShulhan**
[DEMO LINK](https://ivanshulhan.github.io/react_dynamic-list-of-todos/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,25 +1,98 @@
...
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [value, setValue] = useState('');
+  const [sortBy, setSortBy] = useState('');
+  const [randomSort, setRandomSort] = useState(false);
+
+  useEffect(() => {
+    getTodos()
+      .then(res => {
+        setTodos(res);
+      });
```
Comment (aliceblunt8) [inline src/App.tsx:22]:
```suggestion
      .then(setTodos);
```

Comment (aliceblunt8) [reply src/App.tsx:22]: magic ;)

File: src/components/TodoList/TodoList.tsx [hunk 2]
```diff
@@ -1,44 +1,117 @@
...
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
+        </div>
+
+        <div className="TodoList__list-container">
+          {todos.length > 0 ? (
+            <ul className="TodoList__list">
```
Comment (aliceblunt8) [inline src/components/TodoList/TodoList.tsx:70]: TodoList component should contain only the list itself and nothing more, move the form to App component

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (IvanShulhan) [reply src/App.tsx:22]: )

File: src/components/TodoList/TodoList.tsx (hunk 2 above)
Comment (IvanShulhan) [reply src/components/TodoList/TodoList.tsx:70]: Ok, don't prfoblem, but in tasks was something like this:
image

**Review by Bohdan-Maksymiuk — APPROVED** [review]

**Pull request closed without merging**